src/eventgrid/azext_eventgrid/tests/latest/test_eventgrid_commands.py

```diff
@@ -1082,6 +1082,7 @@ def test_private_link(self, resource_group):
             'topic_name': topic_name,
             'location': 'centraluseuap'
         })
+        self.cmd('group create -n {resource_group_net} -l eastus')
```

Comment:
Why add this line?

Reply:
this resource group is hard-coded in the test. Original user use two resource groups. I don't want to change logic.